crates/fee/src/lib.rs

```diff
@@ -159,14 +160,6 @@ decl_storage! {
                     config.vault_rewards,
                     config.relayer_rewards,
                     config.maintainer_rewards,
-                    config.collator_rewards,
-                ]
-            ).unwrap();
-
-            Pallet::<T>::ensure_rationals_sum_to_one(
-                vec![
-                    config.vault_rewards_issued,
-                    config.vault_rewards_locked,
```

Comment:
Wasn't really sure how to handle this split so I make vault stake proportional to `issued_tokens` + `backing_collateral`.